stable/pipelines-v2/values.yaml

```diff
@@ -1,5 +1,5 @@
 crd:
-  create: false
+  create: true
```

Comment:
This would be controlled by the provisioned so can be rolled back to default value false